packages/@aws-cdk/aws-lambda-event-sources/lib/dynamodb.ts

```diff
@@ -22,10 +24,18 @@ export class DynamoEventSource extends StreamEventSource {
       throw new Error(`DynamoDB Streams must be enabled on the table ${this.table.node.path}`);
     }
 
-    target.addEventSourceMapping(`DynamoDBEventSource:${this.table.node.uniqueId}`,
+    const eventSourceMapping = target.addEventSourceMapping(`DynamoDBEventSource:${this.table.node.uniqueId}`,
       this.enrichMappingOptions({eventSourceArn: this.table.tableStreamArn})
     );
+    this._eventSourceMappingId = eventSourceMapping.eventSourceMappingId;
 
     this.table.grantStreamRead(target);
   }
+
+  /**
+   * The Ref of the EventSourceMapping
+   */
+  public get eventSourceMappingId(): string | undefined {
```

Comment:
We should probably define this as a property on `IEventSource` so that all future event sources also implement.

Reply:
I thought about it, but there are also classes like `SnsEventSource` & `S3EventSource` implement the `IEventSource` interface and do not have an `eventSourceMappingId`. 

Should I still add it to the interface?